packages/task/src/node/task-abstract-line-matcher.ts

```diff
@@ -31,6 +31,7 @@ import { URI as vscodeURI } from '@theia/core/shared/vscode-uri';
 import { Severity } from '@theia/core/lib/common/severity';
 
 const endOfLine: string = isWindows ? '\r\n' : '\n';
+const MAX_RANGE_COLUMN = 2147483647;
```

Comment:
Might it make sense to export something like `MAX_SAFE_INTEGER` from somewhere in `core`?

Reply:
Yeah, makes sense. I added a `numbers.ts` in `core/common`.